### ISSUE TITLE
Installation of xpra-x11 fails on Debian 10

### ISSUE THREAD

**Issue by olifre** [Xpra-org/xpra#4150]
**Describe the bug**
```
Setting up xpra-x11 (5.0.6-r0-1) ...^M
  File "<fstring>", line 1^M
    (content_type=)^M
                 ^^M
SyntaxError: invalid syntax^M
^M
dpkg: error processing package xpra-x11 (--configure):^M
 installed xpra-x11 package post-installation script subprocess returned error exit status 1^M
```

**To Reproduce**
1. Start with a vanilla, minimal Debian 10 container. 
2. Execute:
```
wget -q https://xpra.org/gpg.asc -O- | apt-key add - 
wget -q https://xpra.org/repos/buster/xpra.list -O /etc/apt/sources.list.d/xpra.list
apt-get update

apt-get -y install      xpra \
                        xpra-html5 \
                        websockify \
                        menu-xdg \
                        python3-pyinotify \
                        python3-uinput \
                        hicolor-icon-theme
```

**System Information (please complete the following information):**
This is with Debian 10, trying to install `5.0.6-r0-1` XPRA packages. 

**Additional context**
Debian 10 is using Python 3.7.3-1, maybe there's an incompatibility in some fstring?

**Comment by totaam** [comment]
Already fixed in 8141bcfa20df314f11f7b60d770ee3c6a22aeacd - 5.0.6-r1 will hit the repos soon.
As I am sure you are aware: _Debian 10 has been superseded by [Debian 11 (bullseye)](https://www.debian.org/releases/bullseye/). Security updates have been discontinued as of June 30th, 2022._

**Comment by olifre** [comment]
> Already fixed in [8141bcf](https://github.com/Xpra-org/xpra/commit/8141bcfa20df314f11f7b60d770ee3c6a22aeacd) - 5.0.6-r1 will hit the repos soon.

Many thanks! :+1: 

> As I am sure you are aware: _Debian 10 has been superseded by [Debian 11 (bullseye)](https://www.debian.org/releases/bullseye/). Security updates have been discontinued as of June 30th, 2022._

We are not running it in production, but providing legacy container images. Debian LTS (not the Debian Security team) is still providing [security updates until June 2024](https://wiki.debian.org/LTS) for Debian 10. 
Thanks a lot for still caring about almost-EoL distros (as far as sanely possible).
